File: html/hello.php
<?php
 $colors = array("aqua", "black", "blue", "fuchsia", "gray", "green", "lime", "maroon","navy", "olive", "purple", "red", "silver","teal", "white", "yellow");
date_default_timezone_set('America/Los_Angeles');
 
 header( "Content-type: text/html") ;
 print  "<html><head><title>Hello C Based CGI World</title></head>" ;
 print "<body style='background-color: ".$colors[array_rand($colors)]."'><h1>Hello World from PHP @ ".date("Y-m-d | h:i:sa")."</h1>";
 print "</body></html>";
?>

File: cgi-bin/java/Hello.java
import java.io.*;
import java.util.*;
import java.text.*;

public class Hello
{
  private static final String[] colors = {"aqua", "black", "blue", "fuchsia", "gray", 
    "green", "lime", "maroon", "navy", "olive", "purple", "red", "silver", "teal", 
    "white", "yellow"};

  public static void main (String args[])
  {
    Random random = new Random();
    Date date = new Date();
    DateFormat dateFormat = new SimpleDateFormat("yyyy/MM/dd hh:mm:dd");
    dateFormat.setTimeZone(TimeZone.getTimeZone("America/Los_Angeles"));

    System.out.println("Content-type: text/html\n");
    System.out.println("<!doctype html>");
    System.out.println("<html><head><title>Hello Java Based CGI World</title></head>");
    System.out.println("<body style='background-color: " + 
      colors[random.nextInt(colors.length)] + "'><h1>Hello World from Java @ " + 
      dateFormat.format(date) + "</h1>");
    System.out.println("</body></html>");
  }
}
